sdv/sequential/par.py

```diff
@@ -216,6 +216,9 @@ def _preprocess(self, data):
         preprocessed = super()._preprocess(data)
 
         if self._sequence_index:
+            sequence_index_transformer = self.get_transformers()[self._sequence_index]
+            if sequence_index_transformer.enforce_min_max_values:
+                sequence_index_transformer.enforce_min_max_values = False
```

Comment:
If a user updated the sequence index transformer and set `enforce_min_max_values` to `True`, would this ignore that? It might be better to only change the transformer when it's been automatically assigned, so that way if a user modifies it we don't overwrite their change.

Reply:
Sounds good! I moved it into a overridden `auto_assign_method` so it's always after being auto assigned.